languages/go/reference/README.md

```diff
@@ -73,13 +99,31 @@
 - Nested Functions
 - Partial Application
 - Pipelines (?)
-- Pure Functions (?)
+- Pure Functions
 - Recursion
 - REPL (some community projects)
 - Type Inference
 
+## Advanced
+- Code generation (`go generate`)
+- [plugins](https://golang.org/pkg/plugin/)
```

Comment:
Is this more of a Go ecosystem concept? It sounds to me like it isn't particularly suited for a Concept exercise, although it could be referred to in an `after.md` document e.g.

Reply:
This is for the long game, if we ever get that far. I'd leave it in for now.

Reply:
Okay. IJust keep an eye out that nobody will start creating an exercise for it :)